docs/user_guide/2_software_description.md

```diff
@@ -179,7 +179,12 @@ A simple interface has been provided to control the position of each joint using
   A window with twenty sliders will appear. Moving any slider will cause the corresponding joint on the hand to move. You have to start the hand in either position control or teach mode. If the control is changed, reload the plugin to make sure that the sliders correspont to the control that is running at this moment.
 
 ### Hand Calibration
-This plugin is used internally by Shadow to calibrate the raw data from the position sensors.
+This plugin is used internally by Shadow to calibrate the raw data from the position sensors. The calibration has to be run on the NUC machine, therefore rqt has to be started from it. To do that, you can use an desktop icon prepared for that purpuse or in the console run:
+```
+ssh -o 'StrictHostKeyChecking no' -X user@nuc-control "docker exec -u user -i teleop bash -c 'source /home/user/projects/shadow_robot/base_deps/devel/setup.bash;source /home/user/projects/shadow_robot/base/devel/setup.bash; rqt'"
```

Comment:
Include a picture of the NUC RQT desktop icon. Remove teleop, if you must include the ssh script, get the latest one from aurora